verify.go

```
@@ -112,7 +113,9 @@ func (p7 *PKCS7) UnmarshalSignedAttribute(attributeType asn1.ObjectIdentifier, o
 
 func parseSignedData(data []byte) (*PKCS7, error) {
 	var sd signedData
-	asn1.Unmarshal(data, &sd)
+	if r, err := asn1.Unmarshal(data, &sd); err != nil {
+		log.Println(r, err)
```

should return the error?

---

fixed